### DISCUSSION TITLE
Tenants or private documents for users

### DISCUSSION THREAD

**Discussion by Nils98Ar** [jonaswinkler/paperless-ng#1337] [Support]
Hello,

is it possible to configure something like tenants in paperless-ng or to set permissions that everyone can see only his own documents?

I have multiple users and each of them should manage his or her own documents.
At the moment everyone sees all documents.

Thank you!

Nils

**Comment by arifer612** [comment]
Unfortunately, there doesn't seem to be plans to have this implemented. It's being discussed in https://github.com/jonaswinkler/paperless-ng/discussions/52

I suggest you head over there to add on to the comments or upvote those you agree with.
